webdriver/tests/bidi/browsing_context/create/type.py

```diff
@@ -14,6 +14,9 @@ async def test_type(bidi_session, top_context, type_hint):
     contexts = await bidi_session.browsing_context.get_tree(max_depth=0)
     assert len(contexts) == 1
 
+    # It is not guaranteed that the top_context is focused, so we activate it.
+    await bidi_session.browsing_context.activate(context=top_context["context"])
```

Comment:
Hm, so after each test we clean-up the open windows and tabs. So if this is the one and only remaining tab, why is it not focused?

Reply:
In case of chromedriver, the active can be the Mapper tab.

Reply:
Is that also the case for the very first test or related to our clean-up of the session? To me it feels that we should not have this logic in a single test but in the setup/teardown logic instead, and adding a comment why it is needed so that we can remove it once the mapper tab is gone.